Makefile

```diff
@@ -309,13 +318,24 @@ protogen_local: go_protoc-go-inject-tag ## Generate go structures for all of the
 	$(PROTOC_SHARED) -I=./p2p/types/proto --go_out=./p2p/types ./p2p/types/proto/*.proto
 
 	# IBC
+	make copy_ics23_proto
 	$(PROTOC_SHARED) -I=./ibc/types/proto --go_out=./ibc/types ./ibc/types/proto/*.proto
 
 	# echo "View generated proto files by running: make protogen_show"
 
 # CONSIDERATION: Some proto files contain unused gRPC services so we may need to add the following
 #                if/when we decide to include it: `grpc--go-grpc_opt=paths=source_relative --go-grpc_out=./output/path`
 
+.PHONY: copy_ics23_proto
+copy_ics23_proto:
+	echo "Downloading cosmos/ics23 proto definitions..."
+	curl -s -o ./ibc/types/proto/proofs.proto https://raw.githubusercontent.com/cosmos/ics23/master/proto/cosmos/ics23/v1/proofs.proto && \
```

Comment:
I know both curl and wget get the job done, but I've seen `wget` used more frequently in these types of contexts

Reply:
wget is not installed out of the box on most linux distros as such I think it better to use curl which is part of the coreutils.

Reply:
Noted regarded `coreutils`. I have found `wget` (verified by chatgpt) to be the more common default, but not a block by any means. 
![Screenshot 2023-07-25 at 1 49 27 PM](https://github.com/pokt-network/pocket/assets/1892194/febdb5ff-b996-40cb-b83e-69196913cc73)